docusaurus/docs/user-docs/settings/review-workflows.md

```diff
@@ -63,15 +58,28 @@ To add a new stage in the review workflows:
 1. Click on the **Add new stage** button.
 2. Write the *Stage name*.
 3. Select a *Color*.
-3. Click on the **Save** button.
+4. Select *Roles* that can change the stage, if the entity is currently in that review stage.
+5. Click on the **Save** button.
 
 By default new stages are appended, but they can be reordered anytime using the ![drag & drop](/img/assets/icons/drag.svg) button.
 
+:::note
+Setting up roles for each stage can be a tedious task. You can either click "Apply to all stages", to apply the current roles to all other stages
```

Comment:
```suggestion
To set up roles for each stage, you can either click "Apply to all stages" to apply the current roles to all other stages of the workflow or use "Duplicate stage" of the stage context menu.
```

We should not say something is easy, difficult, tedious, etc. This remains subjective and the documentation should stay neutral and non-judgemental (esp. towards our own product 😅 — I'd say that if something is "tedious", let's fix it product-side rather than document workarounds 😄).